support_for_advertising_use_cases.md

```diff
@@ -154,15 +175,67 @@ These use-cases are more focused on the publisher's perspective.
 
 Advertisers need to know how many conversions happened as a result of their ad campaigns. 
 
-## Lift Measurement
+## Conversion Lift Measurement
 
 Ideally advertisers would like to know how many conversions were caused by their ad campaign (i.e. would not have happened were it not for this ad campaign). This is variously refered to as "causality" or "incrementality" measurement. The gold standard measurement approach to answer this question is “Lift Measurement”. This involves a “test group” who is shown ads, and a “control group” who is not shown ads. These groups should be randomly selected prior to running the test to ensure they are well balanced. 
 
 The total number of raw conversion events is counted in both groups, and the difference between the two is called the “lift”. If this is a statistically significant value, and the test and control groups were selected randomly, the only explanation for the difference is the causal effect of the ads.
 
+## Attitudinal Lift Measurement
+
+Much like direct response advertisers want to know how many incremental conversions were caused by their ad campaigns, 
+brand advertisers want to know whether their ad campaigns succeeded in making consumers more aware of, interested in, or
+favorable towards their brand, product, or service.
+
+Because these attitudinal outcomes are not linked to a discrete observable action (like an online purchase), they typically 
+must instead be measured through surveys that ask questions like "how likely are you to purchase a Toyota next time you're
+in the market for a car?"
+
+Lift is measured by comparing the prevalence of a given answer among an "exposed" group of respondents who have seen a 
+particular ad campaign compared to the prevalence of that same answer among a control group who have not seen the campaign.
+
+When possible, the exposed and control groups should be randomly selected prior to running the test to ensure they are well balanced.
```

Comment:
I should maybe drop this paragraph, but I'm hoping to clarify why Facebook's proposal of adding a single bit for 'exposed vs control' in the browser would not be sufficient for our statistical process, unless advertisers magically start doing RCT's.